features/step_definitions/implicit_flow_steps.rb

```diff
@@ -0,0 +1,40 @@
+o_auth_query_params = %i(
+  redirect_uri=http://localhost:3000
+  response_type=id_token%20token
+  scope=openid%20read
+  nonce=hello
+  state=hi
+).join("&")
+
+Given /^I send a post request from that client to the implicit flow authorization endpoint$/ do
+  client_json = JSON.parse(last_response.body)
+  visit new_openid_connect_authorization_path + "?client_id=#{client_json["o_auth_application"]["client_id"]}&#{o_auth_query_params}"
```

Comment:
Line is too long. [133/120]<br>Prefer single-quoted strings inside interpolations.